modules/deploy/pages/deployment-option/cloud/api/cloud-controlplane-api.adoc

```diff
@@ -0,0 +1,184 @@
+= Manage Clusters using the Cloud API
+:description: Use the Cloud API to manage your clusters and organization-level resources in Redpanda Cloud.
+:page-cloud: true
+
+The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud. The Control Plane API enables you to programmatically manage your clusters outside of the Redpanda Cloud UI. You can call the API endpoints directly, or use tools like Terraform or Python scripts to automate cluster management.
+
+== Control Plane API
+
+The Control Plane API is one central API that allows you to provision clusters and networks.
+
+The following services consist of Control Plane API endpoints:
+
+* ClusterService
+* NamespaceService
+* OperationService
+* NetworkService
+
```

Comment:
Let's add a small section on Rate Limiting for Control Plane API. Do you agree @birdayz

Reply:
i would like to bring rate limiting only once we're going to GA. at the moment, the threshold is so high, that people will never hit it, unless it's really abuse.